crates/reth/evm/src/utils.rs

```diff
@@ -22,7 +23,11 @@ pub fn wei_to_sats(wei: U256) -> (U256, U256) {
 }
 
 /// Collects withdrawal intents from bridge-out events in the receipts.
-/// Returns a vector of `WithdrawalIntent`.
+/// Returns a vector of optional [`WithdrawalIntent`]s.
```

Comment:
```suggestion
/// Returns a vector of [`WithdrawalIntent`]s.
```

Reply:
Done in d0f094cb9b2f8aee51466c6ff3848d99f98730b9